forte/mrdsrg-spin-integrated/master_mrdsrg.h

```diff
@@ -90,6 +90,9 @@ class MASTER_DSRG : public DynamicCorrelationSolver {
     /// Return the Hbar of a given order
     std::vector<ambit::Tensor> Hbar(int n);
 
+    /// Return the Hbar0
```

Comment:
Elaborate comment? Why do we need this for the spin-integrated code but not for the spin-adapted one?

Reply:
Both is not needed, this function is for tests ... I simply forgot to remove them.

Reply:
Removed.